src/components/scenes/SendScene.js

```diff
@@ -324,7 +319,9 @@ class SendComponent extends React.PureComponent<Props, State> {
   }
 
   renderSelectedWallet() {
-    const { lockInputs, lockTilesMap = {} } = this.props
+    const { lockInputs } = this.props
+    const { lockTilesMap = {} } = this.props.route.params
+
```

Comment:
Delete this newline. Let's do all the destructuring in one big block at the top of each method.